### PR TITLE
Remove mentions of v1alpha5

### PR DESCRIPTION
**Pull request by mquhuy**

### CODE REVIEW

File: lib/common.sh [hunk 1]
```diff
@@ -251,7 +249,7 @@ export IRONIC_NAMESPACE=${IRONIC_NAMESPACE:-"baremetal-operator-system"}
 export NAMEPREFIX=${NAMEPREFIX:-"baremetal-operator"}
 
 # CAPM3 and IPAM controller images
-if [ "${CAPM3RELEASEBRANCH}" == "release-0.5" ] || [ "${CAPM3_VERSION}" == "v1alpha5" ]; then
+if [ "${CAPM3RELEASEBRANCH}" == "release-0.5" ]; then
```
Comment (tuminoid) [inline lib/common.sh:252]: `==` -> `=`

File: lib/releases.sh [hunk 2]
```diff
@@ -49,7 +49,7 @@ CAPIRELEASEPATH="{https://api.github.com/repos/${CAPI_BASE_URL:-kubernetes-sigs/
 BMORELEASEPATH="{https://api.github.com/repos/${BMO_BASE_URL:-metal3-io/baremetal-operator}/releases}"
 
 # CAPM3, CAPI and BMO releases
-if [ "${CAPM3RELEASEBRANCH}" == "release-0.5" ] || [ "${CAPM3_VERSION}" == "v1alpha5" ]; then
+if [ "${CAPM3RELEASEBRANCH}" == "release-0.5" ]; then
```
Comment (tuminoid) [inline lib/releases.sh:52]: `==` -> `=`

**Review by tuminoid — COMMENTED** [review]
Fix the `==` -> `=` while you're touching those lines.

**Review by mboukhalfa — CHANGES_REQUESTED** [review]
From the previous upgrade trigger it seems removing all related v1alpha5 will break upgrade test which does upgrade from v1a5 to v1b1 can you check what exactly the upgrade test requires from dev-env

**Comment by mquhuy** [comment]
> From the previous upgrade trigger it seems removing all related v1alpha5 will break upgrade test which does upgrade from v1a5 to v1b1 can you check what exactly the upgrade test requires from dev-env

It actually passed in the latest run. I will run it one last time, just to make sure.

File: lib/common.sh [hunk 3]
```diff
@@ -112,12 +112,10 @@ function clone_repo() {
...
 
 # Configure common environment variables
-CAPM3_VERSION_LIST="v1alpha5 v1beta1"
+CAPM3_VERSION_LIST="v1beta1"
 export CAPM3_VERSION="${CAPM3_VERSION:-"v1beta1"}"
 
-if [ "${CAPM3_VERSION}" == "v1alpha5" ]; then
-  export CAPI_VERSION="v1alpha4"
-elif [ "${CAPM3_VERSION}" == "v1beta1" ]; then
+if [ "${CAPM3_VERSION}" == "v1beta1" ]; then
```
[the remark this answers is not in the document]
Comment (mquhuy) [reply lib/common.sh]: Done

File: lib/common.sh (hunk 1 above)
Comment (mquhuy) [reply lib/common.sh]: Done

File: lib/releases.sh (hunk 2 above)
Comment (mquhuy) [reply lib/releases.sh:52]: Done

**Review by kashifest — DISMISSED** [review]

**Review by tuminoid — DISMISSED** [review]
/lgtm

**Comment by tuminoid** [comment]
OK, tide is stuck again, let me test a new theory how to unblock it.

/test all

**Review by kashifest — APPROVED** [review]

File: lib/common.sh [hunk 4]
```diff
@@ -251,16 +249,16 @@ export IRONIC_NAMESPACE=${IRONIC_NAMESPACE:-"baremetal-operator-system"}
 export NAMEPREFIX=${NAMEPREFIX:-"baremetal-operator"}
 
 # CAPM3 and IPAM controller images
-if [ "${CAPM3RELEASEBRANCH}" == "release-0.5" ] || [ "${CAPM3_VERSION}" == "v1alpha5" ]; then
+if [[ "${CAPM3RELEASEBRANCH}" == "release-0.5" ]]; then
```
Comment (mboukhalfa) [inline lib/common.sh:252]: As v1alpha5 and release-0.5 point to the same thing while release-0.5 still needed by upgrade just thinking what in some places we could clean it as well  ?

Comment (mquhuy) [reply lib/common.sh:252]: I'd vote to merge this one first, and address that issue in another PR, as it's likely to require changes in other repos as well. How about it?

**Review by mboukhalfa — APPROVED** [review]
let's follow up with more cleaning related to deprecating v1alpha5

**Pull request merged**